lib/app/audio.nit

```diff
@@ -0,0 +1,42 @@
+# this file is part of NIT ( http://www.nitlanguage.org ).
+#
+# Copyright 2014 Romain Chanoir <romain.chanoir@viacesi.fr>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# App audio abstraction
+module audio
+
+
+# This is an abstraction of a sound that can be played by any apps
+abstract class Sound
+
+	# The resource ID of this sound
+	var id: nullable Int
+
+	# Plays the sound
+	fun play is abstract
+
+	# Pauses the sound
+	fun pause is abstract
+
+	# Resumes the sound
+	fun resume is abstract
+end
+
+
```

Comment:
Space. So much space. Gotta love space.

Seriously though, one space is enough between two entities.